pnpm-lock.yaml

```diff
@@ -26287,6 +26401,38 @@ packages:
       - zod
     dev: false
 
+  /wagmi@1.3.4(encoding@0.1.13)(react-dom@18.2.0)(react@18.2.0)(typescript@5.1.3)(viem@1.2.9):
```

Comment:
Pls downgrade the pnpm version to 8.6.0 and rerun pnpm install

Reply:
yep willl now